Khiem_Nguyen/Python_3/Python/python_stack/python_OOP/zoo.py

```diff
@@ -0,0 +1,18 @@
+class Zoo:
+    def __init__(self, zoo_name):
+        self.animals = []
+        self.name = zoo_name
+    def addDog(self, name):
+        self.animals.append( Dog(name) )
+    def addDragon(self, name):
+        self.animals.append( Dragon(name) )
+    def printAllHealth(self):
+        print("-"*30, self.name, "-"*30)
+        for animal in self.animals:
+            animal.displayHealth()
```

Comment:
where is displayHeath() coming from?

Reply:
This was not an assignment. It was just a file that I created and used some of the code to try to understand how a function can have multiple arguments. It introduced to us *splat and lead into the math dojo assignment.

This was the text for the module
"Note that we have a property/attribute in class Zoo called animals, which contains a list of objects from different classes.  Just as you can store a string, a number, or even an anonymous function inside a list, you can also store objects/instances of another class into the list.  Isn't this cool?

(note: in order to have the code above work, you need to also have class Dog and class Dragon defined above)"

Reply:
ok. I got it.